Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -132,7 +132,18 @@ namespace AzFramework
         ClaimEntitiesCommand queueEntry;
         queueEntry.m_ticketId = ticket.GetId();
         queueEntry.m_listCallback = AZStd::move(listCallback);
-        QueueRequest(ticket, optionalArgs.m_priority, AZStd::move(queueEntry));
+        QueueRequest(&GetTicketPayload<Ticket>(ticket), optionalArgs.m_priority, AZStd::move(queueEntry));
+    }
+
+    void SpawnableEntitiesManager::ClaimEntity(
+        AZ::EntityId entityId, void* ticket,
+        ClaimEntityOptionalArgs optionalArgs)
+    {
+        AZ_Assert(ticket == nullptr, "Ticket provided to ClaimEntity is invalid.");
```

Comment:
should this return if ticket is invalid? I don't think Assert returns in release builds

Reply:
No, because an assert should crash and if the ticket is null then it will cause a crash as well down the road. The assert is here to identify that crash earlier on so it's easier to fix the underlying bug.